src/meta/src/hummock/compaction/manual_compaction_picker.rs

```diff
@@ -61,7 +157,7 @@ impl CompactionPicker for ManualCompactionPicker {
         tmp_sst_info.key_range = Some(self.option.key_range.clone());
         range_overlap_info.update(&tmp_sst_info);
 
-        let level_table_infos: Vec<SstableInfo> = levels[level]
+        let level_table_infos: Vec<SstableInfo> = levels.levels[level - 1]
```

Comment:
Can we add a util function to extract level / sub-level  by level idx from the levels proto instead of explicitly doing `level-1` when getting the level? It is misleading and very easy for people to forget that `levels.levels` is starting from L1.

Reply:
done.